README.md

```diff
@@ -100,6 +100,50 @@ client.set_pool_info("autoscale",
     )
 ```
 
+**Configuration**
+An AzureClient object can be instantiated and initialized with pool, mounted containers and container registries using a configuration file. This is especially useful if the same pool will be used for running multiple batch jobs and experiments. Use the following example to create a configuration file:
+
+```
+[Authentication]
+subscription_id="REPLACE_WITH_AZURE_SUBSCRIPTION_ID"
+resource_group="EXT-EDAV-CFA-PRD"
+user_assigned_identity="REPLACE_WITH_USER_ASSIGNED_ID"
+tenant_id="REPLACE_WITH_TENANT_ID"
+batch_application_id="REPLACE_WITH_BATCH_APP_ID"
+batch_object_id="REPLACE_WITH_BATCH_OBJECT_ID"
+sp_application_id="REPLACE_WITH_SERVICE_PRINCIPAL_APP_ID"
+vault_url="https://cfa-common-adf-prd.vault.azure.net/"
+vault_sp_secret_id="REPLACE_WITH_SECRET_ID"
+subnet_id="REPLACE_WITH_AZURE_SUBNET_ID"
+
+[Batch]
+batch_account_name="cfaprdba"
+batch_service_url="https://cfaprdba.eastus.batch.azure.com"

```

Comment:
change this to a generic batch_service_url since it could change from user to user